app/manage.py

```diff
@@ -51,8 +52,15 @@ def run(self):
             #         db.session.commit()
 
             # todo: [Nick] refactor this, create full seed script
-            print("~~~~~~~~~~ Searching for float wallet ~~~~~~~~~~")
+            print("~~~~~~~~~~ Searching for master organisation ~~~~~~~~~~")
```

Comment:
we now have a system level organisation - this will be used for settings in cases where a user isn't explicity assigned to an organisation

Reply:
that makes sense

Reply:
Cool